tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -465,6 +232,11 @@
     <SuppressFromVsix Include="System.Xml.XPath.XDocument.dll" />
   </ItemGroup>
   <!-- Workaround for https://github.com/dotnet/sdk/issues/433 -->
```

Comment:
It is. The itemdefinition group is part of the workaround. I'll add a comment to document the 'end'